### DISCUSSION TITLE
Support of modify close/minimize/zoom button positions?

### DISCUSSION THREAD

**Discussion by sarukomine** [NativePHP/laravel#304] [Ideas]
I want to add a custom title bar with 40px height, but close/minimize/zoom button positions are not centered vertically.

<img width="912" alt="螢幕截圖 2024-05-15 13 21 02" src="https://github.com/NativePHP/laravel/assets/39088208/b12fa59a-7940-430e-b0bf-6851d7e8f32f">

So I think it will be great if NativePHP have function to change these positions.

**Comment by simonhamp** [comment]
It [seems to be possible](https://www.electronjs.org/docs/latest/tutorial/window-customization#customize-the-traffic-light-position-macos) with Electron.

I haven't got that far in yet, but I don't think we've currently got support for this... someone else raised a [similar request](https://github.com/orgs/NativePHP/discussions/282) some time ago...

**Reply by sarukomine** [reply]
I have submitted a new PR for this feature, you are free to review at anytime~ thanks
